ci/azure-pipelines.yml

```diff
@@ -23,8 +23,40 @@ resources:
       image: rdmz-harbor.rdmz.labs.mlnx/xpmem/x86_64/ubuntu20.04:1
     - container: ubuntu22
       image: rdmz-harbor.rdmz.labs.mlnx/xpmem/x86_64/ubuntu22.04:1
+    - container: fedora
+      image: rdmz-harbor.rdmz.labs.mlnx/ucx/fedora33:1
 
 stages:
+  - stage: Codestyle
+    jobs:
+      - job: format
+        displayName: Format code
+        pool:
+          name: MLNX
+          demands:
+          - ucx_docker
+        container: fedora
+        steps:
+          - checkout: self
+            clean: true
+            fetchDepth: 100
+          - bash: |
+              set -x
+              BASE_SOURCEVERSION=$(git rev-parse HEAD^)
```

Comment:
for my understanding, this would also work in case of multiple patches present in a given PR?

Reply:
Sure, whatever the commit contains.